### PR TITLE
Migrated infra to support node >= 6

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5897 @@
+{
```
Comment (mickhansen) [inline package-lock.json]: No reason to have a lock file i believe?

**Comment by amilajack** [comment]
Also there aren't any tests for this repo. Can you manually test these changes locally?

**Comment by mickhansen** [comment]
What do you mean there aren't any tests?
There are both tests and travis :)

**Comment by amilajack** [comment]
Ahh never mind. I was confusing this project with another one

File: .babelrc [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
-  "plugins": ["transform-object-rest-spread"],
+  "plugins": [
+    "transform-object-rest-spread"
+  ],
   "presets": [
-    "async-to-bluebird",
-    "es2015-node4"
+    ["env", {
+      "targets": {
+        "node": 6
```
Comment (mickhansen) [inline .babelrc:8]: think we could probably go 8 and then drop async to bluebird

**Pull request closed without merging**